doc/source/dev/contributor/quickstart_gitpod.rst

```diff
@@ -0,0 +1,156 @@
+*  `VSCode autodocstrings extension <https://marketplace.visualstudio.com/items?itemName=njpwerner.autodocstring>`_ 
+*  vim
+
+Development workflow
+-----------------------
+The  :ref:`development-workflow` section of this documentation contains information regarding the SciPy development workflow. Make sure to check this before working on your contributions.
+
+When using Gitpod, note these main differences with the setup described in :ref:`development-workflow`.
+
+#. You do not need to configure your git username, and email as this should be done for you as you authenticated through GitHub. You can check the git configuration with the command ``git config --list`` in your terminal.
+#. As you started your workspace from your own SciPy fork, you will by default have both "upstream "and "origin "added as remotes. You can verify this by typing ``git remote`` on your terminal or by clicking on the **branch name** on the status bar (see image below).
+
+.. image:: ../../_static/gitpod/scipy-gitpod-branches.png 
+    :alt: Gitpod workspace branches plugin screenshot
+
+Rendering the Scipy documentation
+----------------------------------
+You can find the detailed documentation on how rendering the documentation with Sphinx works in the :ref:`rendering-documentation` section.
+
+The documentation is pre-built during your workspace initialization. So once this task is completed, you have two main options to render the documentation in Gitpod.
+
+Option 1: Using Liveserve 
+***************************
+
+#. View the documentation in ``scipy/doc/build/html-scipyorg``. You can start with "index.html "and browse, or you can jump straight to the file you're interested in.
+#. To see the rendered version of a page, you can right-click on the ``.html`` file and click on **Open with Live Serve**. Alternatively, you can open the file in the editor and click on the **Go live** button on the status bar.
+    .. image:: ../../_static/gitpod/vscode-statusbar.png 
+        :alt: Gitpod workspace VSCode start live serve screenshot
+#. A simple browser will open to the right-hand side of the editor. We recommend closing it and click on the **Open in browser** button in the pop-up.
+#. To stop the server click on the **Port: 5500** button on the status bar.
+
+Option 2: Using the rst extension
+***********************************
+
+A quick and easy way to see live changes in an `rst` as you work on it uses the rst extension with docutils. 
+
+.. note:: This will generate a simple live preview of the document without the ``HTML`` theme, and some backlinks might not be added correctly. But it is an easy and lightweight way to get instant feedback on your work.
+
+#. Open any of the source documentation files located in ``doc/source`` in the editor.
+#. Open VSCode Command Palette with :kbd:`Cmd-Shift-P` in Mac or :kbd:`Ctrl-Shift-P` in Linux and Windows. Start typing "restructured" and choose either "Open preview" or "Open preview to the Side".
```

Comment:
ahh pretty! learned something new here, never seen `:kbd:`

Reply:
Strangely enough this whole sentence and the next one are made bold though:

<img width="546" alt="image" src="https://user-images.githubusercontent.com/98330/115123155-1fbbc700-9fbc-11eb-818f-7a8a582d58f0.png">

Unclear what's going on there.

Reply:
Maybe because of the `.. image::` inside a numbered list?